src/argilla/server/security/model.py

```diff
@@ -62,7 +62,7 @@ class UserCreate(BaseModel):
 class UserGetter(GetterDict):
     def get(self, key: str, default: Any = None) -> Any:
         if key == "full_name":
-            return f"{self._obj.first_name} {self._obj.last_name}"
+            return " ".join(filter(None, [self._obj.first_name, self._obj.last_name]))
```

Comment:
I see a lot of logic just to prevent a default value. Wdyt @jfcalvo?

```suggestion
            return f"{self._obj.first_name} {self._obj.last_name or ''}".strip()
```

Reply:
I think using `strip` is a bit misleading. If we want to really simplify then I would go with something like this:

```suggestion
            return f"{self._obj.first_name} {self._obj.last_name}" if self._obj.last_name else self._obj.first_name
```